smokes/e2e/about.scenarios.coffee

```diff
@@ -3,14 +3,26 @@
 # to use previous and next link
 
 aboutPage = require('./pages/about.coffee')
+homePage = require('./pages/home.coffee')
+builderPage = require('./pages/builder.coffee')
 
-describe('', () ->
+describe 'about test', () ->
     about = null
+    home =  null
+    builder = null
 
-    beforeEach(() ->
+    beforeEach () ->
         about = new aboutPage('runtests')
-    )
+        builder = new builderPage('runtests', 'force')
+        home = new homePage()
+        builder.goDefault()
+        home.getAnonymousButton().click()
+        home.setUserText("homer")
+        home.setPasswordText("doh!")
+        home.getLoginButton().click()
 
+    afterEach () ->
+        home.setLogOut()
```

Comment:
you don't set anything there, you just logOut, so just call that

    home.logOut()